src/cljfx/fx/fill_transition.clj

```diff
@@ -0,0 +1,24 @@
+(ns cljfx.fx.fill-transition
+  "Part of a public API"
+  (:require [cljfx.composite :as composite]
+            [cljfx.lifecycle :as lifecycle]
+            [cljfx.coerce :as coerce]
+            [cljfx.fx.transition :as fx.transition])
+  (:import [javafx.animation FillTransition]))
+
+(set! *warn-on-reflection* true)
+
+(def props
+  (merge
+    fx.transition/props
+    (composite/props FillTransition
+      :shape [:setter lifecycle/dynamic]
+      :duration [:setter lifecycle/scalar :coerce coerce/duration :default 400]
+      :from-value [:setter lifecycle/scalar :coerce coerce/color :default nil]
+      :to-value [:setter lifecycle/scalar :coerce coerce/color :default nil])))
```

Comment:
should remove `:default`s: `nil` is not a valid input for `coerce/color`

Reply:
Done.